index.html

```diff
@@ -41,83 +54,124 @@ <h4 class="connect">LET'S CONNECT</h4>
+                    </ul>
+                    <p class="paragraph">
+                        A daily selection of privately personalised reads; no accounts or no sign-ups required.
+                    </p>
+                    <ul class="langs">
+                        <li class="lang1 languages">html</li>
+                        <li class="lang2 languages">css</li>
+                        <li class="lang3 languages">javascript</li>
+                    </ul>
+                    <button type="button" class="seeProject disabled" disabled>See Project</button>
+                </div>
+                
+            </div>
+            <!-- card 3 -->
+            <div class="card">
+                <div class="media">
+                    <img class="image" src="./Images/Snapshoot-page-4.png" alt="tonic">
+                </div>
+                <div class="card-content">
+                    <h2>Tonic</h2>
+                    <ul class="triple">
+                        <li class="canopy">CANOPY</li>
+                        <li class="double">Back End Dev</li>
+                        <li class="double">2015</li>
+                    </ul>
+                    <p class="paragraph">
+                        A daily selection of privately personalised reads; no accounts or no sign-ups required.
+                    </p>
+                    <ul class="langs">
+                        <li class="lang1 languages">html</li>
+                        <li class="lang2 languages">css</li>
+                        <li class="lang3 languages">javascript</li>
+                    </ul>
+                    <button type="button" class="seeProject trois pressed">See Project</button>
+                </div>
+                
+            </div>
+            <!-- card 4 -->
+            <div class="card multi-post">
+                <div class="media">
+                    <img class="image" src="./Images/Snapshoot-page-5.png" alt="tonic">
+                </div>
+                <div class="card-content">
+                    <h2 class="dimi">Multi-Post Stories</h2>
+                    <ul class="triple cc">
+                        <li class="canopy">CANOPY</li>
+                        <li class="double">Back End Dev</li>
+                        <li class="double">2015</li>
+                    </ul>
+                    <p class="paragraph">
+                        A daily selection of privately personalised reads; no accounts or no sign-ups required.
+                    </p>
+                    <ul class="langs">
+                        <li class="lang1 languages">html</li>
+                        <li class="lang2 languages">css</li>
+                        <li class="lang3 languages">javascript</li>
+                    </ul>
+                    <button type="button" class="seeProject">See Project</button>
+                </div>
+                
+            </div>
+        </section>
+    </div>
+    <section class="About Myself">
```

Comment:
- [x] Great job working on changes requested by the previous reviewer, But unfortunately, you need to stick to the design as much as possible for better UI (user interface) and this is a good professional practice that every frontend software developer should maintain. Please, make sure to layout this section `About section`  so they can look closely similar to the [original design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471) 😸 .

You can use `flex-direction: column;` in the CSS what do you think

## **Your project**
![real](https://user-images.githubusercontent.com/80895497/150654358-5f39dad3-9e79-438a-aa12-bd7d626dc198.PNG)

## **Design**
![design](https://user-images.githubusercontent.com/80895497/150654359-cd0f1ebe-0d77-41e2-82a6-64af01a54cdf.PNG)